src/components/EditorsColumn.jsx

```diff
@@ -76,8 +77,12 @@ export default class EditorsColumn extends React.Component {
           style={{flex: editorsFlex[index]}}
           onHide={partial(onComponentHide, `editor.${language}`)}
           onRef={partial(this._storeEditorRef, index)}
+          onToggleEditorTextSize={partial(onToggleEditorTextSize,
+           `editor.${language}`)}
```

Comment:
From a product standpoint I think it would make the most sense to have a single size control for all the editors—I can’t think of a situation in which you’d want different editors at different sizes. WDYT?